infra/build_deploy_config.py

```diff
@@ -89,8 +110,16 @@
     "GCP_DEFAULT_REGION",
 ]
 
+if not validate_aws_config():
+    exit(1)
+
+caller_info = boto3.client("sts").get_caller_identity()
+owner = caller_info['UserId'].split(':')[1]
```

Comment:
@kislyuk  this would allow the `role_session_name` to be populated into owner; might need to update travis_test credentials with a `role_session_name` value within the CD.

Reply:
In our system, infrastructure changes are always performed by an operator. Travis or Allspark should not be updating these tags.

Reply:
Noted: ensures operator has role_session_name defined :)